Project/frontend/app/get-started/page.tsx

```diff
@@ -22,7 +22,7 @@ const GetStarted = () => {
         <Text size="md">This is a blank page for new users.</Text>
 
         {/* Dicsord Login Link*/}
-        <Button component={Link} href="/get-started" className={classes.loginButton} mt="xl" justify="center"> Login with Discord </Button>
+        <Button component={Link} href="https://discord.com/oauth2/authorize?client_id=1292671408026943532&response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A3000%2Fprofile&scope=identify" className={classes.loginButton} mt="xl" justify="center"> Login with Discord </Button>
```

Comment:
remove this hard coded link for now until we actually start working on it

Reply:
Ok removed